examples/components/CCFRAUD_VERTICAL/traininsilo/aml_comm.py

```diff
@@ -0,0 +1,365 @@
+from abc import ABC
```

Comment:
Exact same file as the MNIST one, right? Is that not possible to check it in just once, and have both components refer to the same file?

Reply:
Yes, unfortunately afaik SDK v2 does not support external includes as of now.